mantidproject/mantid
# Layout improvements to MuonAnalysis

Two changes to the layout of MuonAnalysis as requested by scientists.
The changes here are cosmetic only, no functionality has been altered.

1. On the Data Analysis tab, the excess blank space in the Data and Settings sections has been removed and given to the Function section. The boundaries between each section is user-adjustable by dragging a splitter. This enables more space to be given to the fit function, making it easier to see everything without scrolling.
2. On the Results Table tab, expand the drop-down list from the combo boxes to show the full label names (useful for long names) and remove the unnecessary word "fits" to give more space.

**To test:**

1. *Data Analysis tab layout*
  - Open *Interfaces/Muon/Muon Analysis* and load any muon data
  - On Settings tab, ensure `Enable multiple fitting` is checked
  - Go to Data Analysis tab and compare the layout to issue #18059. Verify that, by dragging the splitters if necessary, there is more space for the fit function.

Old            |  New
:-------------------------:|:-------------------------:
![verticalspace](https://cloud.githubusercontent.com/assets/15363125/20427556/892c6352-ad7c-11e6-8491-6c34c07ded74.png)   | ![newverticalspace](https://cloud.githubusercontent.com/assets/15363125/20427561/8c45324e-ad7c-11e6-95a6-691c8aa529e0.png)

(Note: the Workspace Index field has been removed because it was unnecessary, it is always zero)

2. *Results tab combo boxes*
  - Load all the files from `\\olympic\babylon5\Scratch\TomPerkins\combobox_test_label` into MantidPlot (i.e. MantidPlot itself, not the muon interface). This saves having to actually run all the fits.
![workspaces](https://cloud.githubusercontent.com/assets/15363125/20427477/153c877e-ad7c-11e6-9203-1af4d3a87aa5.PNG)

  - Go to the Results Table tab of MuonAnalysis
  - In the lower, Fitting Results section, look at the combo boxes for *Sequential* and *Simultaneous* fits. Compare to issue #18062:

Old            |  New
:-------------------------:|:-------------------------:
![dropdown_label](https://cloud.githubusercontent.com/assets/15363125/20427719/35df2cba-ad7d-11e6-884d-6c318ab14b04.png) | ![dropdown_new](https://cloud.githubusercontent.com/assets/15363125/20427722/37559642-ad7d-11e6-9573-9fc793c2cd17.png)


Fixes #18059 
Fixes #18062 

Screenshots in the documentation and testing guide have been updated where necessary.

[Release notes](https://github.com/mantidproject/mantid/blob/f7c137d99a72e2fd6a9f791b7d67ed95de7cb8e5/docs/source/release/v3.9.0/muon.rst#id1) have been updated.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
